src/app/tabs/app-web3-provider.js

```diff
@@ -0,0 +1,31 @@
+import { Plugin } from '@remixproject/engine'
+import * as packageJson from '../../../package.json'
```

Comment:
Consider using the destructuration synthax to improve tree shaking when enabled : 
`import { version } from '...'`